tasks/backfill_gh_app_installations.py

```diff
@@ -54,6 +60,28 @@ def add_repos_service_ids_from_provider(
             gh_app_installation.repository_service_ids = new_repo_service_ids
             db_session.commit()
 
+    def is_gh_app_remote_selection_all(
+        self,
+        db_session: Session,
+        owner_service,
+        gh_app_installation: GithubAppInstallation,
+    ):
+        remote_gh_app_installation = async_to_sync(
+            owner_service.get_gh_app_installation
+        )(installation_id=gh_app_installation.installation_id)
+        repository_selection = remote_gh_app_installation.get(
+            "repository_selection", ""
+        )
+        if repository_selection == "all":
+            gh_app_installation.repository_service_ids = None
+            db_session.commit()
```

Comment:
i wouldn't expect this side effect from a function named `is_condition_true()`, consider renaming to something like `maybe_set_installation_to_all_repos()`